bot/exts/evergreen/duck_game.py

```diff
@@ -0,0 +1,329 @@
+        """Create and send the initial game embed. This will be edited as the game goes on."""
+        image = assemble_board_image(game.board, game.rows, game.columns)
+        with BytesIO() as image_stream:
+            image.save(image_stream, format="png")
+            image_stream.seek(0)
+            file = discord.File(fp=image_stream, filename="board.png")
+        embed = discord.Embed(
+            title="Duck Duck Duck Goose!",
+            color=Colours.bright_green,
+            footer=""
+        )
+        embed.set_image(url="attachment://board.png")
+        return await ctx.send(embed=embed, file=file)
+
+    async def display_claimed_answer(self, game: DuckGame, author: discord.Member, answer: tuple[int]) -> None:
+        """Add a claimed answer to the game embed."""
+        async with game.editing_embed:
+            game_embed, = game.embed_msg.embeds
+            old_footer = game_embed.footer.text
+            if old_footer == discord.Embed.Empty:
+                old_footer = ""
+            game_embed.set_footer(text=f"{old_footer}\n{str(answer):12s}  -  {author.display_name}")
+            await self.edit_embed_with_image(game.embed_msg, game_embed)
+
+    async def end_game(self, game: DuckGame, end_message: str) -> None:
+        """Edit the game embed to reflect the end of the game and mark the game as not running."""
+        game.running = False
+
+        scores = sorted(
+            game.scores.items(),
+            key=lambda item: item[1],
+            reverse=True,
+        )
+        scoreboard = "\n".join(f"{member.display_name}: {score}" for member, score in scores)
+
+        missed = [ans for ans in game.solutions if ans not in game.claimed_answers]
+        if missed:
+            missed_text = "Flights everyone missed:\n" + "\n".join(f"{ans}" for ans in missed)
+        else:
+            missed_text = "All the flights were found!"
+
+        game_embed, = game.embed_msg.embeds
+        old_footer = game_embed.footer.text
+        if old_footer == discord.Embed.Empty:
+            old_footer = ""
+
+        embed_as_dict = game_embed.to_dict()  # Cannot set embed color after initialization
+        embed_as_dict["color"] = discord.Color.red().value
+        game_embed = discord.Embed.from_dict(embed_as_dict)
+
+        game_embed.set_footer(
+            text=f"{old_footer.rstrip()}\n\n{end_message} Here are the scores:\n{scoreboard}\n\n{missed_text}"
+        )
+        await self.edit_embed_with_image(game.embed_msg, game_embed)
+
+    @start_game.command(name="help")
+    async def show_rules(self, ctx: commands.Context) -> None:
+        """Explain the rules of the game."""
+        await self.send_help_embed(ctx)
+
+    @start_game.command(name="stop")
+    async def stop_game(self, ctx: commands.Context) -> None:
+        """Stop a currently running game. Only available to mods."""
+        if not any(role in ctx.author.roles for role in MODERATION_ROLES):
```

Comment:
You can use the [`with_role`](https://github.com/python-discord/sir-lancebot/blob/e6e280cd13236647ffcaa35f522baeb747fbf97e/bot/utils/decorators.py#L147) decorator instead of this check. Example of it's usage [here](https://github.com/python-discord/sir-lancebot/blob/e6e280cd13236647ffcaa35f522baeb747fbf97e/bot/exts/internal_eval/_internal_eval.py#L153). 

Also, I'm not sure if this current code would work as `MODERATION_ROLES` contains id's (ints) but you're comparing against a list of role objects (ctx.author.roles)